src/vivarium_public_health/metrics/reporters.py

```diff
@@ -26,6 +26,18 @@ def name(self) -> str:
 COLUMNS = __Columns()
 
 
+def write_dataframe(
+    results: pd.DataFrame,
+    measure: str,
+    results_dir: Optional[Union[str, Path]],
+) -> None:
+    """Utility function for observation 'report' methods to write pd.DataFrames to parquet"""
+    if results_dir is None:
+        raise ValueError("A results_dir must be specified to write out results.")
+    results.to_parquet(Path(results_dir) / f"{measure}.parquet", index=False)
+
+
+# FIXME: remove this when ready
```

Comment:
I need to update the other components to use the new `write_dataframe` method and then I'll remove this